Plot graph without edges

When i tried to plot a graph without an edge, i got the following assertion error.

```
assert len(edges) > 0, "The list of edges has to be non-empty."
AssertionError: The list of edges has to be non-empty
```

This means that Netgraph has a condition that prevents it from plotting, so this condition needs to be removed.

--- a/requirements.txt
+++ b/requirements.txt
@@ -12,7 +12,7 @@ grandalf==0.7
 grinpy==19.7a0
 kiwisolver==1.4.4
 matplotlib==3.6.2
-netgraph==4.10.2
+netgraph==4.12.6
 network2tikz==0.1.8
 networkx==2.8.8
 numpy==1.23.4

